app/models/game.rb

```diff
@@ -51,4 +51,26 @@ def populate_board! # rubocop:disable Metrics/AbcSize
 
     pieces.create(color: 'black', type: 'King', current_row_index: 7, current_column_index: 4)
   end
+
+  def forfeit(current_user_id)
+    current_user = User.find(current_user_id)
+    if current_user.id == white_player_id
+      if black_player_id.nil?
+        update_attributes(winner_id: game.black_player_id)
+        other_user = User.find(black_player_id)
```

Comment:
other_user is the same here as `black_player`. So doing the `find` isn't necessary